### PR TITLE
Make ccm great(installable as package) again

### PR DESCRIPTION
**Pull request by fruch**
* added the psutil that was missing
* run shell script were missing when used as installed package

### CODE REVIEW

**Comment by fruch** [comment]
taking it for a spin in BYO:
https://jenkins.scylladb.com/job/scylla-master/job/manual-and-scheduled-tests/job/byo_build_tests_dtest/589/console

File: ccmlib/scylla_node.py [hunk 1]
```diff
@@ -637,7 +637,7 @@ def import_bin_files(self):
                                                  'scylla-jmx'))
 
         parent_dir = os.path.dirname(os.path.realpath(__file__))
-        resources_bin_dir = os.path.join(parent_dir, '..', 'resources', 'bin')
+        resources_bin_dir = os.path.join(parent_dir, 'resources', 'bin')
```
Comment (slivne) [inline ccmlib/scylla_node.py]: how come this works ?

Comment (fruch) [reply ccmlib/scylla_node.py]: The resources folder is inside ccm, moving into ccmlib and making is sure it's part of the installation doesn't change the behavior.

**Comment by slivne** [comment]
Ok - you can merge if BYO says it works

**Comment by fruch** [comment]
BYO said yes

**Pull request merged**